docker/templates/Dockerfile.erb

```
@@ -96,7 +82,10 @@ RUN addgroup -g 1000 logstash && \
   find /usr/share/logstash -type d -exec chmod g+s {} \; && \
   ln -s /usr/share/logstash /opt/logstash
 
-COPY --from=builder-env2yaml /tmp/go/src/env2yaml/env2yaml /usr/local/bin/env2yaml
+# Copy env2yaml from tarball (pre-compiled by Gradle)
```

no need to have this comment in final dockerfile

```suggestion
<%# Copy env2yaml from tarball (pre-compiled by Gradle) %>
```